### PR TITLE
WIP Increment total_people_joined count for team

### PR DESCRIPTION
**Pull request by davidxia**
WORK IN PROGRESS. DO NOT MERGE.

This increments a count on the Team model to track
how many people have signed up via a team page.

### CODE REVIEW

**Comment by davidxia** [comment]
@aaronlifshin @jtolds, @brad, @jwhite66

Incrementing the team's total_people_joined attribute now works! I also realized that the join form fails to convey to users why they should join/signup and what they get in return.

Please take a look at the latest WIP commit above. I left TODO notes in the commit message and in the code/templates to address the lack explaining what "join" is to the user.

**Comment by davidxia** [comment]
FYI I will be gone on vacation without Internet for a week starting tomorrow night until Oct 6 :(

**Comment by aaronlifshin** [comment]
OK, should I got ahead and merge?

**Comment by davidxia** [comment]
@aaronlifshin No, it's still WIP. Somebody who knows better the MayDay talking points and messaging should find all the "TODO"s in this commit and actually write real text instead of the lorem ipsum stuff I put in there.

File: templates/join_team_success.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends "includes/base.html" %}
+
+{% block body %}
+  <div class="page-body">
+    <h4>You've successfully joined the MayDay PAC's mailing list.</h4>
+
+    <!-- TODO (davidxia) Fix up content and styling of this page. -->
```
Comment (davidxia) [inline templates/join_team_success.html:7]: @aaronlifshin Here's an example.

File: templates/show_team2.html [hunk 2]
```diff
@@ -180,19 +179,37 @@ <h5 class="pledges" id="team_total_pledges"></h5>
           <!-- join form !-->
           <div class="panel panel-default">
             <div class="panel-heading">
+              <!-- TODO (davidxia) Someone who knows MayDay's official talking points and messaging should fill in the appropriate text below. -->
```
Comment (davidxia) [inline templates/show_team2.html:182]: @aaronlifshin This is the other major place.

**Pull request closed without merging**